guides/common/modules/ref_glossary-of-terms-used-in-project.adoc

```diff
@@ -422,7 +422,7 @@ The files contain code to define a set of necessary resources, such as packages,
 Do not confuse with xref:Manifest[Manifest (Red{nbsp}Hat subscription manifest)].
 
 [[Puppet_server]]
-Puppet server:: A {SmartProxyServer} component that provides Puppet Manifests to hosts for execution by the Puppet Agent.
+Puppet server:: A {SmartProxyServer} component that provides a Puppet catalog to hosts for execution by the Puppet Agent.
```

Comment:
```suggestion
Puppet server:: A {SmartProxyServer} component that provides a Puppet catalog to hosts for execution by the Puppet agent.
```
I don't think this has to be capitalized.

Reply:
@ekohl If you make this small change, I think it should be fine for merging.

Reply:
Then I think it should also be done in other places, so I added a commit that does that first.